pset-functions.js

```diff
@@ -115,6 +171,60 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+// function isPalindrome(word) {
+//   let i = 0
+//   let letters = word.split('')
+//   console.log(letters)
+//   for (let i = 0; i < letters.length; i++) {
+//     let currentLetter = letters[i]
+//     let lastLetter = letters[(letters.length - 1 - i)]
+//     console.log(i)
+//     console.log(letters.length - i)
+//     if (currentLetter === lastLetter) {
+//       return true
+//       console.log(currentLetter)
+//     }
+//     else {
+//       return false
+//       console.log(currentLetter)
+//       console.log(lastLetter)
+//     }
+//   }
+// }
+
+function isPalindrome(word) {
+  let letters = word.split('')
+  let lettersReverse = letters.slice(0).reverse()
```

Comment:
I'm not really sure the ".slice(0)" portion is affecting the code, but using the for loop is definitely a creative way to solve the problem! Here's another potential solution that takes advantage of a conditional returning a true/false value [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L174).